src/mono/System.Private.CoreLib/src/System/Threading/ThreadPool.Browser.Mono.cs

```diff
@@ -16,7 +16,7 @@ namespace System.Threading
 #if !FEATURE_WASM_THREADS
     [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
 #endif
-    public sealed class RegisteredWaitHandle : MarshalByRefObject
+    public sealed partial class RegisteredWaitHandle : MarshalByRefObject
```

Comment:
I don't think this should be necessary, there might have been an issue before with some files getting incorrectly included when `FeaturePortableThreadPool == false`